packages/security/carbon.yml

```diff
@@ -0,0 +1,313 @@
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-components-stringformatter
+      tags:
+        - content-element
+  time-indicator:
+      name: Time indicator
+      status: stable
+      type: component
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/time-indicator.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-components-timeindicator
+      tags:
+        - system-feedback
+        - content-element 
+  truncated-list:
+      name: Truncated list
+      status: stable
+      type: component
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/truncated-list.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-components-truncatedlist
+      tags:
+        - data-display
+        - content-element
+  type-layout:
+      name: Type layout
+      status: stable
+      type: component
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/type-layout.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-components-typelayout
+      tags:
+        - data-display
+        - content-element
+  combo-button:
+      name: Combo button
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/combo-button.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-combobutton
+      tags:
+        - input-control
+  filter-panel:
+      name: Filter panel
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/filter-panel.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-filterpanel
+      tags:
+        - form
+  icon-button-bar:
+      name: Icon button bar
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/icon-button-bar.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-iconbuttonbar
+      tags:
+        - input-control
+  non-entitled-section:
+      name: Non entitled section
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/non-entitled-section.svg'
```

Comment:
missing thumb. do patterns not have thumbnails? if that's the case let's remove the "thumbnailPath" property